assets/src/components/reporting/ReportingDashboard.tsx

```diff
@@ -183,13 +205,23 @@ class ReportingDashboard extends React.Component<Props, State> {
     );
   };
 
+  formatResponseTimeStats = (responseTime: number) => {
+    const seconds = Math.round(responseTime);
```

Comment:
nit: is this necessary?

Reply:
This is slightly more accurate than if we didn't have it. If it is 25.8 seconds without it it would become 25 seconds instead of 26 seconds. Its pretty simple piece of code and makes it more accurate so I think we should just keep it in